tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+This tutorial introduces the basics for Sentinel-1 use in GEE. It will illustrate basic SAR terminology and demonstrate data selection and visualization.
+In a next tutorial, we'll introduce more advanced concepts.
+
+## SAR characteristics
+
+A good introduction into Synthetic Aperture Radar (SAR) basics is [A Tutorial on Synthetic Aperture Radar](https://elib.dlr.de/82313/) by a group of experts at DLR led by Dr. A. Moreira. The essentials are in part I and II. Advanced polarimetry and interferometry, or combined POLINSAR, (part III *ff*) is currently outside the scope of what is possible in GEE.
+
+A key point about SAR is that it is a radar (RAdio Detection And Ranging) instrument, and it's basic measurements are intensity and phase measurements of the backscattered signal, sampled in time bins along the **azimuth** (across track direction of the sensor antenna) and **range** (perpendicular to the direction of the sensor antenna). These time bins relate to locations on Earth from which the backscattered signal originate. The "Synthetic Aperture" of the SAR is the virtual creation of a huge antenna which makes use of the sensor motion and the associated complex data processing, which is necessary to create high resolution in azimuth. By moving along the orbit, the next azimuth line is created from a new microwave pulse in the range direction (this is oversimplified).
+
+The key difference between SAR and optical sensors is that
+
+- SAR is an **active** sensor, transmitting **microwave** radiation, for which it receives the portion **scattered back** to the sensor. Optical sensors are **passive** sensors that register reflected radiation from the Sun (in specific parts of the visible and infrared spectrum).
+- SAR is **side-looking**, unlike optical sensors which are, usually, nadir looking. 
+- SAR radiation is **coherent**, within a pre-set range of wavelengths. This is useful, because it allows precise **phase** and **intensity** measurements and the use of **polarization**. However, it is also the cause of **speckle**. Optical sensors are not coherent, apart from laser and lidar instruments. 
+
+Reflecting on these differences is important, because it is the basis to understand the relative advantages of SAR compared to optical sensors:
+
+- SAR does not depend on Sun light, thus works **day and night**, provided the sensor acquires data (i.e. is switched on)
```

Comment:
SAR instruments do not depend on sunlight, this they can produce data **day and night**, provided the sensor is on.

Reply:
revised